### PR TITLE
Remove the api.Node.rootNode entry

### PR DESCRIPTION
**Pull request by foolip**
This was in Gecko, but quickly reverted and renamed to getRootNode():
https://github.com/mozilla/gecko-dev/commit/4967a99f0cf3814469f8ef5f33d142f586d640b6
https://github.com/mozilla/gecko-dev/commit/9adeb967f0a1f1bfe15f927ea5b7201233eedf98
https://github.com/mozilla/gecko-dev/commit/9404a1745dd91937edf7c0be89f1cfc34d2ea252

https://developer.mozilla.org/en-US/docs/Web/API/Node/rootNode
should be redirected to getRootNode instead.

### CODE REVIEW

**Comment by Elchi3** [comment]
> https://developer.mozilla.org/en-US/docs/Web/API/Node/rootNode
> should be redirected to getRootNode instead.

Done and also removed `rootNode from` the `Node` main page.

**Review by Elchi3 — APPROVED** [review]
LGTM, unsure if I should merge given the failing CI atm.

**Comment by foolip** [comment]
It was the labeled that failed, which doesn't really matter, but I'll try to get it passing before merging for good measure.

**Comment by foolip** [comment]
Ok, all sorted.

**Pull request merged**
